<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmoothDialog</class>
 <widget class="QDialog" name="SmoothDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smooth</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/CartoGrapher/resources/m.svg</normaloff>:/plugins/CartoGrapher/resources/m.svg</iconset>
  </property>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="output_checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Open output file after running algorithm</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Iteration numbers:</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="layerCbmBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>160</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsFileWidget" name="output_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input layer or shp：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_iteration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothed layer：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
